### PR TITLE
chore(localdebug): output env in local tunnel task

### PR DESCRIPTION
**Pull request by XiaofuHuang**
local tunnel task: 
```
        {
            "label": "Start local tunnel",
            "type": "teamsfx",
            "command": "debug-start-local-tunnel",
            "args": {
                "ngrokArgs": "http 3978",
                "env": "local",
            }
        }
```

Output env:
```
TUNNEL_ENDPOINT=https://xxxx-xxxx-xxxx-xxxx-xx-xxxx-xxxx-xxxx-xxxx.ngrok.io
TUNNEL_DOMAIN=xxxx-xxxx-xxxx-xxxx-xx-xxxx-xxxx-xxxx-xxxx..ngrok.io
```
Failure output:
image

### CODE REVIEW

File: packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts [hunk 1]
```diff
@@ -45,6 +54,10 @@ const ngrokEndpointRegex =
   /obj=tunnels name=(?<tunnelName>.*) addr=(?<src>.*) url=(?<endpoint>https:\/\/([\S])*)/;
 const ngrokWebServiceRegex = /msg="starting web service" obj=web addr=(?<webServiceUrl>([\S])*)/;
 const defaultNgrokWebServiceUrl = "http://127.0.0.1:4040/api/tunnels";
+const outputName = {
+  endpoint: "TUNNEL_ENDPOINT",
+  domain: "TUNNEL_DOMAIN",
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts:59]: Then when to transfer to `BOT_DOMAIN` to be referenced in manifest?

Comment (XiaofuHuang) [reply packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts:59]: Can we use ` env/generate`?
```
  - uses: env/generate # Generate env to .env file
    with:
      envs:
        BOT_DOMAIN: ${{TUNNEL_DOMAIN}}
        BOT_ENDPOINT: ${{TUNNEL_ENDPOINT}}
```

Comment (swatDong) [reply packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts:50]: Anyway, work for happy path and collect feedback later.

**Review by swatDong — APPROVED** [review]

File: packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts (hunk 1 above)
Comment (kuojianlu) [reply packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts:50]: Any concern to not directly use `BOT_DOMAIN` and `BOT_ENDPOINT`?

**Review by tecton — APPROVED** [review]

**Pull request merged**